Source/missiles.cpp

```diff
@@ -1616,10 +1616,6 @@ void AddFireNova(Missile &missile, Point dst, Direction midir)
 	UpdateMissileVelocity(missile, dst, sp);
 	SetMissDir(missile, GetDirection16(missile.position.start, dst));
 	missile._mirange = 256;
-	missile.var1 = missile.position.start.x;
-	missile.var2 = missile.position.start.y;
-	missile.var4 = missile.position.start.x;
-	missile.var5 = missile.position.start.y;

```

Comment:
Can you elaborate why we can remove this? I don't see it being compensated anywhere else in the PR.

Reply:
Never used by `MI_FireNova`